src/core/namespace.c

```diff
@@ -3210,8 +3241,10 @@ static const char *const protect_system_table[_PROTECT_SYSTEM_MAX] = {
 DEFINE_STRING_TABLE_LOOKUP_WITH_BOOLEAN(protect_system, ProtectSystem, PROTECT_SYSTEM_YES);
 
 static const char *const protect_control_groups_table[_PROTECT_CONTROL_GROUPS_MAX] = {
-        [PROTECT_CONTROL_GROUPS_NO]     = "no",
-        [PROTECT_CONTROL_GROUPS_YES]    = "yes",
+        [PROTECT_CONTROL_GROUPS_NO]      = "no",
+        [PROTECT_CONTROL_GROUPS_YES]     = "yes",
```

Comment:
Not a matter, but then I'd like to the first two entries are aligned in the first commit, that is, please insert one more space in the first commit.